src/main/java/com/rarchives/ripme/ripper/rippers/JabArchivesRipper.java

```diff
@@ -33,7 +33,7 @@ public String getDomain() {
 
     @Override
     public String getGID(URL url) throws MalformedURLException {
-        Pattern p = Pattern.compile("^https?://jabarchives.com/main/view/([a-zA-Z0-9_]+).*$");
+        Pattern p = Pattern.compile("^https?://(:?www.)jabarchives.com/main/view/([a-zA-Z0-9_]+).*$");
```

Comment:
Shouldn't this be `(?:www\\.)?` instead of `(:?www.)`

Reply:
Yea it should